lms/static/js/groups/views/cohort_editor.js

```diff
@@ -18,11 +27,38 @@
         render: function() {
             this.$el.html(this.template({
                 cohort: this.model,
-                advanced_settings_url: this.advanced_settings_url
+                contentGroups: this.contentGroups,
+                studioAdvancedSettingsUrl: this.context.studioAdvancedSettingsUrl
             }));
+            this.cohortFormView = new CohortFormView({
+                model: this.model,
+                contentGroups: this.contentGroups,
+                context: this.context
```

Comment:
What I decided to do was pass a context to each view rather than having to manually pass around all the different URLs that are needed. Even though this view currently only uses one item from the context, I prefer to be consistent about passing the whole context through.

Note that line 31 is different as that is a rendering of a template. I decided to not pass the context to the templates but to instead leave the URLs passed in directly. I'd be willing to change them to all receive a context instead if that seems more consistent.